tests/utils.py

```diff
@@ -0,0 +1,28 @@
+import importlib
+import unittest
+
+
+def is_transformers_available():
+    # return importlib.util.find_spec("transformers") is not None
+    try:
+        import transformers
+
+        is_available = True
+    except:
+        is_available = False
+    return is_available
+
+
+def is_torch_available():
+    return importlib.util.find_spec("torch") is not None
```

Comment:
Why does this implementation differ from the one above for transformers? Can it be reused?

Reply:
Yes, it should be re-usable. Problem was happening because of some error in my system. Will fix it.

Reply:
Because of some error, I am unable to use `importlib.util.find_spec(...)` for `transformers`. So, will stick to `import transformers` way only.